webhook_server/libs/handlers/runner_handler.py

```
@@ -735,6 +735,90 @@ async def is_branch_exists(self, branch: str) -> bool:
+                return False
+
+            # Amend the commit author to match the original PR's Signed-off-by
+            author_spec = f"{author_name} <{author_email}>"
+            rc, _, err = await run_command(
+                command=f"{git_cmd} commit --amend --no-edit --author={shlex.quote(author_spec)}",
+                log_prefix=self.log_prefix,
+                redact_secrets=[github_token, author_spec, author_email, author_name],
+                mask_sensitive=self.github_webhook.mask_sensitive,
+            )
+            if not rc:
+                self.logger.warning(f"{self.log_prefix} Failed to amend cherry-pick author for DCO compliance: {err}")
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)